pcsx2/Docs/GameIndex.md

```diff
@@ -124,28 +124,41 @@ The clamp modes are also numerically based.
 
-## GS hardware fixes
-
-*   autoFlush
-*   conservativeFramebuffer
-*   cpuFramebufferConversion
-*   disableDepthSupport
-*   wrapGSMem
-*   preloadFrameData
-*   fastTextureInvalidation
-*   textureInsideRT
-*   alignSprite
-*   mergeSprite
-*   wildArmsHack
-*   pointListPalette
-*   mipmap
-*   trilinearFiltering
-*   skipDrawStart
-*   skipDrawEnd
-*   halfBottomOverride
-*   halfPixelOffset
-*   roundSprite
-*   texturePreloading
+## GS Hardware Fixes
+
+### GS Hardware Mipmap Fixes
+
+*   mipmap                     [`0` or `1` or `2`]    (Off, Basic, Full)                    Default: Automatic (No value)
+*   trilinearFiltering         [`0` or `1` or `2`]    (None, Trilinear, Trilinear Ultra)    Default: None (`0`)
+
+### GS Hardware General Fixes
+
+*   conservativeFramebuffer   [`0` or `1`]            (Off or On)                           Default: On (`1`)
+*   texturePreloading         [`0` or `1` or `2`]     (None, Partial or Full Hash Cache)    Default: None (`0`)
+
+### GS Hardware Renderer Fixes
+
+*   autoFlush                [`0` or `1`]   (Off or On)                          Default: Off (`0`)
+*   disableDepthSupport      [`0` or `1`]   (Off or On)                          Default: Off (`0`)
+//  Disable Safe Features - Too aggressive for GameDB.
+*   fastTextureInvalidation  [`0` or `1`]   (Off or On)                          Default: Off (`0`)
+*   cpuFramebufferConversion [`0` or `1`]   (Off or On)                          Default: Off (`0`)
+*   wrapGSMem                [`0` or `1`]   (Off or On)                          Default: Off (`0`)
+*   preloadFrameData         [`0` or `1`]   (Off or On)                          Default: Off (`0`)
+*   textureInsideRT          [`0` or `1`]   (Off or On)                          Default: Off (`0`)
+*   halfBottomOverride       [`0` or `1`]   (Force-Disabled or Force-Enabled)    Default: Automatic (No value)
+*   pointListPalette         [`0` or `1`]   (Off or On)                          Default: Off (`0`)
+
+### GS Hardware Upscaling Fixes
+
+*   alignSprite              [`0` or `1`]                      (Off or On)       Default: Off (`0`)
+*   mergeSprite              [`0` or `1`]                      (Off or On)       Default: Off (`0`)
+*   wildArmsHack             [`0` or `1`]                      (Off or On)       Default: Off (`0`)
+*   skipDrawStart            [Value between `0` to `100000`]   (0-100000)        Default: Off (`0`)
+*   skipDrawEnd              [Value between `0` to `100000`]   (0-100000)        Default: Off (`0`)
+*   halfPixelOffset          [`0` or `1` or `2` or `3`]   (Off, Normal Vertex, Special Texture or Special Texture Aggressive)    Default: Off (`0`)
+*   roundSprite              [`0` or `1` or `2`]          (Off, Half or Full)    Default: Off (`0`)
+//  Texture Offsets X and Y - Because of the changing values per upscaling multiplier this isn't a good option, look for Half-Pixel Offset or even Wild Arms to reduce ghosting.
```

Comment:
Remove this also, there's no point in mentioning features that aren't there and trying to justify them not being there, we're not in court.